templates/layout.html

```diff
@@ -52,6 +52,17 @@
     </div>
     </nav>
 
+{% with messages = get_flashed_messages(with_categories=true) %}
+    {% if messages %}
+        <!-- note that this is in opposite order, I dont know why -->
+        {% for category, message in messages %}
+            <div class="alert alert-{{ category }}">
+                {{ message }}
+            </div>
```

Comment:
I wonder why we are not following the example in doc? https://flask.palletsprojects.com/en/1.1.x/patterns/flashing/#flashing-with-categories . We don't have to handle the `div`s and the class details by ourselves.

In terms of the order, I think your comment line can be removed. It's decided by the underlying data structure, and the users should manage this.

Reply:
I removed the comment. It was unnecessary. 

I started with the code from the Flask documentation. It did not look good on the app so I decided to use the "alert" classes built into Bootstrap.  

Thanks for the feedback!

Reply:
Yep, I had a try and I get what you meant. 

Cool. All look good. Thanks again, I'm happy to see people work together to make a project better. And I hope this is a helpful experience to you as well :)